content/references/rippled-api/validator-list.md

```diff
@@ -0,0 +1,124 @@
+
+    The validator list contents are signed with a separate cryptographic key, so you can verify their integrity regardless of the TLS certificate used.
+
+**Tip:** Since this request uses the GET method, you can test this request using just the URL bar of your web browser. For example, <https://s1.ripple.com:51235/vl/ED2677ABFFD1B33AC6FBC3062B71F1E8397C1505E1C42C64D11AD1B28FF73F4734> requests Ripple's recommended list from one of Ripple's public servers.
+
+## Response Format
+
+The response has the status code **200 OK** and a JSON object in the message body. The response body is very similar to the format used for validator list sites such as <https://vl.ripple.com/>.
+
+The JSON object has the following fields:
+
+| `Field`          | Value  | Description                                      |
+|:-----------------|:-------|:-------------------------------------------------|
+| `manifest`       | String | The list publisher's [manifest data](#manifest-data), in either base64 or hexadecimal. |
+| `blob`           | String | Base64-encoded JSON data representing the validator list. |
+| `signature`      | String | The signature of the `blob` data, in hexadecimal. |
+| `version`        | Number | The version of the validator list protocol this object uses. The current version is **1**. A higher version number indicates backwards-incompatible changes with a previous version of the validator list protocol. |
+<!--{# This field isn't included in the rippled response, but should be. https://github.com/ripple/rippled/issues/3392
+| `public_key`     | String | The public key used to verify this validator list data, in hexadecimal. This is a 32-byte Ed25519 public key prefixed with the byte `0xED`. |
+#}-->
+
+### Manifest Data
+[[Source]](https://github.com/ripple/rippled/blob/97712107b71a8e2089d2e3fcef9ebf5362951110/src/ripple/app/misc/impl/Manifest.cpp#L43-L66 "Source")
+
+A "manifest" contains information uniquely identifying a person or organization involved in the consensus process, either a **validator** or a **list publisher**. A validator's manifest contains the _public_ information from that [validator's token](run-rippled-as-a-validator.html#3-enable-validation-on-your-rippled-server). A list publisher's manifest provides information about the list publisher. Both are typically encoded to binary in the XRP Ledger's standard [binary serialization format](serialization.html). (There is no standard JSON representation of a manifest.)
+
+One of the main purposes of manifests relates to rotating validator keys. When a validator changes its ephemeral key pair, the validator publishes a new manifest with to share its new ephemeral public key, using the validator's master key pair to sign the manifest to prove its authenticity. A validator uses its ephemeral key pair to sign validations as part of the [consensus process](consensus.html) and uses its master key pair only to sign new manifests and similar.
```

Comment:
Couple typos here: 

```
< publishes a new manifest with to share its new ephemeral public key
---
> publishes a new manifest to share its new ephemeral public key
```


Not sure if "similar" should be removed or something added at the end:

```
< uses its master key pair only to sign new manifests and similar.
---
> uses its master key pair only to sign new manifests.
```

Reply:
With "and similar" I was thinking about special cases like the old domain verification process. But now that we have the new process, I guess it's not really necessary. Instead I added a note about how manifests relate to validator tokens.